governance-proposal/TSC-Policy-Draft.md

```diff
@@ -0,0 +1,31 @@
+## ***DRAFT for Public Comment***
+
+# Node.js Foundation TSC Policy
+
+The Board of Directors of Node.js Foundation has set in place the following policies to govern the operation of the Technical Steering Committee (TSC). These policies relate to release processes, technical scope, business goals, etc. The TSC will, to the best of its ability, adhere to these policies. In cases where the TSC makes a judgment that the goals of Node.js Foundation are better served by making exceptions to these policies, it is expected that the TSC will communicate these exceptions and indicate their reasons to the Board at the next Board meeting.
+
+The scope of projects chosen by the TSC will focus on the subsystems, modules, scripts, tools or infrastructure to support Node.js.
+
+Node.js Foundation will support projects focused on the following:
```

Comment:
I don't know that it has any specific intent. Do you have an alternative suggestion? "Approve", "incorporate" and others seem more definitive/restrictive to me, so support works for me. Certainly open to any suggestions of a better word there.

In general, our projects bring certain development efforts under the umbrella of the foundation, but they also work within development communities that are not under the foundation. E.g. we host OPNFV which is a platform for network functions virtualization. They are essentially taking bits from various projects important to running VNFs and creating a platform by "fitting together" other open source projects. They take parts of OpenStack, OpenDaylight, Linux kernel and KVM, DPDK and other code, they build abstraction layers so various parts can be swapped out and build/test that the platform works for hosting VNFs. While they have some projects under the foundation umbrella, most of the real work they do is upstream in the OpenStack, OpenDaylight, Linux kernel, etc communities. That's certain possible for the Node.js community as well where not everything has to be under the foundation and the TSC can support efforts to work upstream (or side-stream I guess) with other development efforts in the community that are important for the Node.js developers to achieve what they need.

Does this help?